package.json

```diff
@@ -149,7 +149,7 @@
     "analyze": "yarn build -- --stats && webpack-bundle-analyzer build/bundle-stats.json",
     "lint": "prettier --list-different '**/*.{js,ts,md,json}' .eslintrc.json .prettierrc .babelrc && eslint '**/*.{js,md}' --max-warnings 0 && stylelint '**/*.{css,scss}'",
     "lint-fix": "prettier --write '**/*.{js,ts,md,json}' .eslintrc.json .prettierrc .babelrc && eslint --fix '**/*.{js,md}' --max-warnings 0 && stylelint --fix '**/*.{css,scss}'",
-    "build-translations": "yarn run babel --quiet --config-file ./babel.i18next-extract.js 'src/**/*.{js,jsx,ts,tsx}' && yarn lint-fix",
+    "build-translations": "rm -rf .tmpi18ncache || true && mkdir .tmpi18ncache && yarn run babel --config-file ./babel.i18next-extract.js 'src/**/*.{js,jsx,ts,tsx}' --out-dir '.tmpi18ncache/' && rm -rf .tmpi18ncache && yarn lint-fix",
```

Comment:
Output everything to a temporary directory, and then clear that directory :)

Reply:
Looks good, but what if it fails on this step?